nullaway/src/main/java/com/uber/nullaway/NullabilityUtil.java

```diff
@@ -480,7 +480,7 @@ public static boolean mayBeNullFieldFromType(
     return !(symbol.getSimpleName().toString().equals("class")
             || symbol.isEnum()
             || codeAnnotationInfo.isSymbolUnannotated(symbol, config, null))
-        && Nullness.hasNullableAnnotation(symbol, config);
+        && Nullness.hasNullableFieldAnnotation(symbol, config);
```

Comment:
At a first glance, it is not obvious that 'hasNullableFieldAnnotation' includes both @Nullable and @MonotonicNonNull. It sounds like it only includes the @Nullable annotation.

One possibility is to rename it to something like "hasNullabilityFieldAnnotation", so that one does not mistake it to only include @Nullable annotation.

Reply:
Renamed to `hasNullableOrMonotonicNonNullAnnotation` in 1aa9747fb946e7a66a9cc6d09def688f0d393ef5